Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,511 @@
+        m_tabWidget->tabBar()->setEnabled(false);
+        vLayout->addWidget(m_tabWidget);
+
+        QFrame* footerFrame = new QFrame(this);
+        footerFrame->setObjectName("createAGemFooter");
+        m_backNextButtons = new QDialogButtonBox(this);
+        m_backNextButtons->setObjectName("footer");
+        QVBoxLayout* footerLayout = new QVBoxLayout();
+        footerLayout->setContentsMargins(0, 0, 0, 0);
+        footerFrame->setLayout(footerLayout);
+        footerLayout->addWidget(m_backNextButtons);
+        vLayout->addWidget(footerFrame);
+
+        m_backButton = m_backNextButtons->addButton(tr("Back"), QDialogButtonBox::RejectRole);
+        m_backButton->setProperty("secondary", true);
+        m_nextButton = m_backNextButtons->addButton(tr("Next"), QDialogButtonBox::ApplyRole);
+
+        connect(m_tabWidget, &TabWidget::currentChanged, this, &CreateAGemScreen::UpdateNextButtonToCreate);
+        connect(m_backButton, &QPushButton::clicked, this, &CreateAGemScreen::HandleBackButton);
+        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::HandleNextButton);
+
+        setObjectName("createAGemBody");
+        setLayout(vLayout);
+    }
+
+    void CreateAGemScreen::LoadButtonsFromGemTemplatePaths([[maybe_unused]]QVBoxLayout* firstScreen)
+    {
+        m_radioButtonGroup = new QButtonGroup();
+
+        auto templatesResult = PythonBindingsInterface::Get()->GetGemTemplates();
+        if (templatesResult.IsSuccess() && !templatesResult.GetValue().isEmpty())
+        {
+            m_gemTemplates = templatesResult.GetValue();
+            for (int index = 0; index < m_gemTemplates.size(); ++index)
+            {
+                ProjectTemplateInfo gemTemplate = m_gemTemplates.at(index);
+
+                QRadioButton* button = new QRadioButton(gemTemplate.m_displayName);
+                button->setObjectName("createAGem");
+                m_radioButtonGroup->addButton(button, index);
+
+                QLabel* buttonSubtext = new QLabel();
+                buttonSubtext->setObjectName("createAGemRadioButtonSubtext");
+                buttonSubtext->setText(gemTemplate.m_summary);
+                if (index == 0)
+                {
+                    button->click();
+                }
+                firstScreen->addWidget(button);
+                firstScreen->addWidget(buttonSubtext);
+            }
+        }
+    }
+
+    QScrollArea* CreateAGemScreen::CreateFirstScreen()
+    {
+        QScrollArea* firstScreenScrollArea = new QScrollArea(this);
+        firstScreenScrollArea->setWidgetResizable(true);
+        firstScreenScrollArea->setMinimumWidth(660);
+        firstScreenScrollArea->setObjectName("createAGemRightPane");
+        QFrame* firstScreenFrame = new QFrame();
+
+        QWidget* firstScreenScrollWidget = new QWidget();
+        firstScreenScrollArea->setWidget(firstScreenScrollWidget);
+
+        QVBoxLayout* firstScreen = new QVBoxLayout(this);
```

Comment:
ditto here - don't think `this` parent is necessary
```suggestion
        QVBoxLayout* firstScreen = new QVBoxLayout();
```